scripts/hdf5_dump.py

```diff
@@ -233,20 +235,20 @@ def print_header_dict(hdict, clock_speed_hz):
 
 
 def print_trigger_record_header(data_array, clock_speed_hz, k_list_components):
-    print_header_dict(unpack_header(data_array, "TriggerRecord Header"), clock_speed_hz)
+    print_header_dict(unpack_header(data_array, "TriggerRecord Header", 4), clock_speed_hz)
 
     if k_list_components:
         comp_keys = DATA_FORMAT["Component Request"]["keys"]
         comp_unpack_string = DATA_FORMAT["Component Request"]["unpack string"]
-        for i_values in struct.iter_unpack(comp_unpack_string, data_array[56:]):
+        for i_values in struct.iter_unpack(comp_unpack_string, data_array[64:]):
             i_comp = dict(zip(comp_keys, i_values))
             print(80*'-')
             print_header_dict(i_comp, clock_speed_hz)
     return
 
 
 def print_fragment_header(data_array, clock_speed_hz):
-    print_header_dict(unpack_header(data_array, "Fragment Header"), clock_speed_hz)
+    print_header_dict(unpack_header(data_array, "Fragment Header", 5), clock_speed_hz)
     return
```

Comment:
I think it would be better to have something like `FRAGMENT_HEADER_VERSION = 5` at the top of this file.